### PR TITLE
Fix/hsc 617 search results tab facet

### PR DESCRIPTION
**Pull request by Metodi-Matov**

### CODE REVIEW

File: src/HawksearchVue.js [hunk 1]
```diff
@@ -224,6 +224,9 @@ class HawksearchVue {
                             if (pageLoadingActions.includes(action)) {
                                 HawksearchVue.scrollToBeginning(this);
                             }
+                        }).then(() => {
```
Comment (GabrielGeorgiev) [inline src/HawksearchVue.js:227]: This should be in SearchBox.vue, on the subsequent search's dispatchToStore

File: src/HawksearchVue.js [hunk 2]
```diff
@@ -771,18 +774,6 @@ class HawksearchVue {
         return configIndexName || urlIndexName || "";
     }
 
-    static getTabField(store) {
```
Comment (GabrielGeorgiev) [inline src/HawksearchVue.js:774]: We can leave this helper method although it is not used currently. Basically, it is a good way to fetch a tab facet an a reference for fetching other types of facets.

File: src/components/facets/FacetList.vue [hunk 3]
```diff
@@ -6,7 +6,7 @@
     @scroll="onScroll">
 
     <div class="hawk-facet-rail__heading" @click="toggleFacetMobileMenu">
-      {{ $t("Narrow Results") }}
+      {{ $t("Filter By") }}
```
Comment (GabrielGeorgiev) [inline src/components/facets/FacetList.vue:9]: I think the branch needs to be rebased or something. Isn't this update from a previous task?

Comment (Metodi-Matov) [reply src/components/facets/FacetList.vue:9]: This is an update form a previous task.

**Pull request merged**